src/SignalR/clients/csharp/Http.Connections.Client/src/HttpConnectionOptions.cs

```diff
@@ -66,6 +76,31 @@ public HttpConnectionOptions()
             set => _headers = value ?? throw new ArgumentNullException(nameof(value));
         }
 
+        /// <summary>
+        /// Gets or sets the maximum buffer size for data written from the transport before it is
+        /// throttled.
```

Comment:
```suggestion
        /// Gets or sets the maximum buffer size for data read by the application.
```

Really I think this should be called `ReadBufferSize` instead of `TransportMaxBufferSize`, but I know we're trying to match the bad naming on the server.